addons/ui/org.openhab.ui.cometvisu/about_files/ANOTHER-REST-CLIENT_LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
Either about_files should be added to build.properties so that it becomes available in the binary or an external link should be used.
Looking at the MIT license, a link to https://opensource.org/licenses/MIT probably won't do, because the year and copyright holder is expected in the title, so the link should point to the license of the project itself. In case there is a risk that the external link breaks (because the project might not be well structured and often changing), including the license might be the safer option.